frontend/src/lib/getUserById.ts

```diff
@@ -0,0 +1,29 @@
+import { env } from "@env.mjs";
+import { User } from "@portaljs/ckan";
+
+export async function getUserById(id: string) {
```

Comment:
This is a duplicate https://github.com/transport-data/tdc-data-portal/blob/main/frontend/src/utils/user.ts#L13